packages/bpk-docs/src/layouts/DocsLayout/DocsLayout.js

```diff
@@ -20,7 +20,7 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 import SideNavLayout from './../SideNavLayout';
-import NeoSideNav from './../NeoSideNav';
+import NeoSideNavLayout from './../NeoSideNav';
```

Comment:
I renamed `NeoSideNav` to make it more consistent with the naming of the current docs components